rules/net/ddos.yara

```diff
@@ -9,6 +8,8 @@ rule ddos_refs : critical {
     $ref = "TSource Engine Query"
     $ref2 = "ackflood" fullword
     $ref3 = "synflood" fullword
+    // datadog-agent tracer-fentry-debug.o
+    $ignore_ref = /synflood\_\w+/
```

Comment:
IMHO, this is too generic of an ignore, and will result in missing hits. Can you find something more specific to datadog-agent - maybe the exact phrase used for synflood_xxx?

Also, please use {0,256} instead of +, as unbounded matches have severe performance impacts in YARA.

Reply:
Will do -- I'll go back and limit any other unbounded changes in the other open PRs as well.